packages/terra-form-field/src/Field.jsx

```diff
@@ -166,6 +166,18 @@ const Field = (props, { intl }) => {
       ) : null
   );
 
+  const content = React.Children.map(children, (child) => {
+    if (required) {
+      if (child && (child.type.isInput || child.type.isTextarea || child.type.isTextarea)) {
```

Comment:
Could remove one level of return by combining the if statement.
```suggestion
      if (required && child && (child.type.isInput || child.type.isTextarea || child.type.isTextarea)) {
```

Reply:
Good call, updated: https://github.com/cerner/terra-core/pull/2398/commits/b6ddd566ed60a1452c19b1b72cfbd652d094eacf